Content.Server/Zombies/ZombifyOnDeathSystem.cs

```diff
@@ -101,7 +107,7 @@ public void ZombifyEntity(EntityUid target, MobStateComponent? mobState = null)
             //we need to basically remove all of these because zombies shouldn't
             //get diseases, breath, be thirst, be hungry, or die in space
             RemComp<RespiratorComponent>(target);
-            RemComp<BarotraumaComponent>(target);
+            // RemComp<BarotraumaComponent>(target); Actually yes, they should die in space (to prevent zombie spacing)
```

Comment:
```suggestion
             RemComp<BarotraumaComponent>(target);
```
Please uncomment this.

Reply:
The vacuum DoT is mostly balanced by the zombie heal over time. Are you sure?

Reply:
Done.